src/azure-cli/azure/cli/command_modules/util/_params.py

```diff
@@ -21,7 +21,8 @@ def load_arguments(self, _):
         c.argument('headers', nargs='+',
                    help="Space-separated headers in KEY=VALUE format or JSON string. Use @{file} to load from a file")
         c.argument('uri_parameters', options_list=['--uri-parameters', '--url-parameters'], nargs='+',
-                   help='Space-separated queries in KEY=VALUE format or JSON string. Use @{file} to load from a file')
+                   help='Query parameters in the URL/URI. Space-separated queries in KEY=VALUE format or JSON string. '
```

Comment:
`URI` is only used for backward compatibility. 

```suggestion
                   help='Query parameters in the URL. Space-separated queries in KEY=VALUE format or JSON string. '
```